django_filters/fields.py

```diff
@@ -17,6 +17,15 @@
     RangeWidget,
 )
 
+try:
+    from django.utils.choices import ChoiceIterator as BaseChoiceIterator
```

Comment:
@carltongibson A heads up that I've opened https://github.com/django/django/pull/17223 to rename `ChoiceIterator` to `BaseChoiceIterator`. Will update this PR to simplify this once that's been merged.